datafusion/sqllogictest/test_files/window.slt

```diff
@@ -4341,6 +4341,9 @@ LIMIT 5;
 24 31
 14 94
 
+statement ok
```

Comment:
I think this one might be better off as batch_size = 1 to avoid the plan changes later in this file

I see the challenge however.